### PR TITLE
fix(TUI-358): moderate CVE on bootstrap sass

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

Security issue reported by github.

**What is the chosen solution to this problem?**

Update bootstrap sass to 3.4.1

diff theme typographie:
image

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/scripts/package.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "uglifyjs-webpack-plugin": "1.2.7",
     "url-loader": "^1.1.2",
     "webpack": "^4.19.0",
-    "webpack-bundle-analyzer": "^3.0.4",
+    "webpack-bundle-analyzer": "^2.9.0",
```
Comment (annilsson) [inline packages/scripts/package.json]: Why downgrade here?

File: packages/scripts/package.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
     "file-loader": "^2.0.0",
     "html-webpack-plugin": "3.2.0",
     "jest": "^24.0.0",
-    "jest-environment-jsdom": "^24.0.0",
+    "jest-environment-jsdom": "^23.4.0",
```
Comment (annilsson) [inline packages/scripts/package.json]: Might want to align with @jsomsanith here: https://github.com/Talend/ui/pull/2059
And update one or the other PR

Comment (jsomsanith) [reply packages/scripts/package.json]: Good catch, no pb, we'll have a conflict :D

File: packages/scripts/package.json (hunk 1 above)
Comment (jsomsanith) [reply packages/scripts/package.json]: +1, why ?

Comment (jsomsanith) [reply packages/scripts/package.json]: +1, why ?

Comment (jmfrancois) [reply packages/scripts/package.json]: this comes from the version script

**Pull request merged**